src/routes/package-edit.js

```diff
@@ -121,6 +121,13 @@ class PackageEditRoute extends Component {
     }
   };
 
+  addPackageToHoldings = () => {
```

Comment:
It would be awesome if we didn't have to duplicate this. How would we share the model between the package show and edit routes? /cc @taras @cherewaty 

Seems like we should at least include a note for future folks looking to make architectural improvements.

Reply:
I've added a comment for now. Would appreciate any recommendations to avoid duplication @taras or @cherewaty. Don't believe I can move to a common utilities method since model and update are needed by the method